x-pack/metricbeat/module/statsd/server/data.go

```diff
@@ -44,15 +44,15 @@ func splitTags(rawTags, kvSep []byte) map[string]string {
 	for _, kv := range tagSplit {
 		kvSplit := bytes.SplitN(kv, kvSep, 2)
 		if len(kvSplit) != 2 {
-			logger.Warn("could not parse tags")
+			log.Named("statd").Warn("could not parse tags")
```

Comment:
```suggestion
			log.Named("statsd").Warn("could not parse tags")
```

not sure, but that's the name of the directory

Reply:
there's a few of these in this file

Reply:
the original logger has `statd` name. Do you want me to change it to `statsd`? 

https://github.com/khushijain21/beats/blob/main/x-pack/metricbeat/module/statsd/server/registry.go#L17

Reply:
Nope, thought it was a typo.